Library/Formula/xorg.rb

```diff
@@ -0,0 +1,470 @@
+class Xorg < Formula
+  desc "Xorg Libraries"
+  homepage "http://www.linuxfromscratch.org/blfs/view/svn/x/x7lib.html"
+  revision 7
```

Comment:
I don't think we need this. Quoting from the [Formula Cookbook](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#formula-cookbook):

> Occasionally, these updates require a forced-recompile of the formula itself or its dependents to either ensure formulae continue to function as expected or to close a security issue. This forced-recompile is known as a revision and inserted underneath the homepage/url/sha block.